### DISCUSSION TITLE
In Trans translation texts, html tags are converted to <0>, <1> ... etc. How does lingui know what tag is actually being used?

### DISCUSSION THREAD

**Discussion by crazyyi** [lingui/js-lingui#1497] [General]
I just found out if I put
`<Trans>This is a <Link href="#">Clickable</Link></Trans>`

or 
```
<Trans>
  <p> - Good point 1 </p>
  <p> - Good point 2 </p>
  <p> - Good point 3 </p>
  <p> - Good point 4 </p>
<Trans>
```

both Link tag and p tag would be converted to <0>, <1>, <2>, ... etc. in the translation texts.

But when switching language, if I don't modify the <0>, <1>, <2> tags, how does lingui know which tag (e.g. Link, p) should be used there?

**Comment by timofei-iatsenko** [comment]
It match them by the index of appearance
